examples/layers/rendering/spinning_square.dart

```diff
@@ -9,6 +9,13 @@ import 'dart:math' as math;
 
 import 'package:flutter/animation.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+
+class NonStopVSync implements TickerProvider {
```

Comment:
I was going to make this change but actually that name makes it sound like it fires continually, which is a bit weird...